### ISSUE TITLE
Getting design choices right: Low latency vs. High privacy

### ISSUE THREAD

**Issue by robertkiel** [w3f/messaging#17]
At the moment, I am going over the README and trying to get things right.

Do we really want to have a privacy-preserving messaging protocol that allows streaming?

As far as I know, it is mandatory to add some unpredictable latency to packets that are sent through a mixnet, so in my opinion low-latency streaming contradicts our privacy demands.

**Comment by burdges** [comment]
Agreed.  We should judge streaming out of scope and prioritize latency flexibility instead.  And streamed data is normally not that sensitive anyways.

In principle, there are streaming-like protocols that might kinda work, but the user experience becomes "wait 30 seconds while we buffer enough to ignore the latency", and they'd damage your privacy properties.  

If we do everything else well, then someone could develop a "privately stream out from a hot zone" service, which repackages the stream for more conventional redistribution.  In other words, near real time media backup or exfiltration, not actual streaming.  Not our problem

**Comment by oskarth** [comment]
Discussed during Brussels workshop; close this issue once reflected in the requirements.
